src/ai/backend/manager/data/permission/role.py

```diff
@@ -123,3 +135,13 @@ class UserRoleAssignmentData:
     user_id: uuid.UUID
     role_id: uuid.UUID
     granted_by: Optional[uuid.UUID] = None
+
+
+@dataclass(frozen=True)
+class RoleListResult:
```

Comment:
How about moving this to `types.py` like `notification`?
It will be difficult to locate if the type is in a different place each time